js/components/date.js

```diff
@@ -95,12 +93,15 @@ Fliplet.FormBuilder.field('date', {
   },
   watch: {
     value: function(val) {
-      if (!val && this.autofill !== 'empty') {
-        this.updateValue(moment().format(DATE_FORMAT));
+      if (this.datepicker) {
+        this.datePicker.datepicker('setDate', val);
+      }
+
+      if (this.$v.value.$invalid) {
+        this.highlightError();
       }
 
-      this.highlightError();
-      this.$emit('_input', this.name, val, false, true);
```

Comment:
Why are the `false, true` parameters removed? They were added as part of https://github.com/Fliplet/fliplet-widget-form-builder/pull/409

Reply:
We've removed 'updateValue' usages which potentially were calling a loop of 'emit' events and now we are firing one 'emit' after the component load and one each time after value is updated.
Should we ask QA to check the issue for you previous fix? If yes, please let me know which one :)

Reply:
The bug was https://weboo.atlassian.net/browse/ID-1394 Please make sure QA verifies the bug is not re-introduced.